code/chapter-06/circle-sdf-distances.py

```diff
@@ -23,7 +23,7 @@
     varying vec2 v_position;
     void main()
     {
-        const float epsilon = 0.005;
+        const float epsilon = 0.05;
```

Comment:
Why did you change this value ?

Reply:
Oh... I was playing with the values when following your tutorial (just to see how stuff change), and probably forgot to return them to the original. You can freely ignore these kind of changes.
My bad.

Reply:
Can you put back original values such that I can merge your PR ?